app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -31,7 +31,7 @@ class Api::V2::SubscriptionsController < Api::V2::ApiController
     param :no_overlap, :bool, :desc => N_("Return subscriptions which do not overlap with a currently-attached subscription")
     def index
       collection = scoped_search(
-        index_relation.distinct, :cp_id, :asc, resource_class: Pool, includes: [:subscription])
+        index_relation.distinct, nil, nil, resource_class: Pool, includes: [:subscription], custom_sort: name_sort)
```

Comment:
Does `:name` not work? With the `custom_sort`, the output doesn't show the user what the default sort is:

```json
{
    "organization": {},
    "total": 3,
    "subtotal": 3,
    "page": 1,
    "per_page": 20,
    "error": null,
    "search": null,
    "sort": {
        "by": null,
        "order": null
    },
```

Reply:
it doesn't because there is no 'name' column on the pools.  We need to switch to scoped_search based sorting instead of handling it ourselves.  That is a large and much riskier change though.  I've opened http://projects.theforeman.org/issues/23512 to handle this in the future.